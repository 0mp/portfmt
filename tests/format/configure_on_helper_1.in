SDL_CONFIGURE_ON=	prefix=${PREFIX} mandir=${MANPREFIX}/man \
	exampledir=${EXAMPLESDIR}
