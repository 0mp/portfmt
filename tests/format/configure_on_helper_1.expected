SDL_CONFIGURE_ON=	exampledir=${EXAMPLESDIR} \
			mandir=${MANPREFIX}/man \
			prefix=${PREFIX}
